src/converters/cover.ts

```diff
@@ -104,6 +125,11 @@ class CoverHandler implements ServiceHandler {
       this.positionCurrent = latestPosition;
       this.scaleAndUpdateCurrentPosition(this.positionCurrent);
     }
+    if (this.hasTilt) {
+      if (this.tiltExpose.property in state) {
```

Comment:
If I recap correctly, it was throwing an `can't find property of undefined` error in the combined if statement.
If we refactor to exclude hasTilt and use tiltExpose explicitly as undefined in case of no tilt expose, I will have a look again at how to reduce complexity here

Reply:
I would suppose that something this should work:
```suggestion
   if (this.tiltExpose !== undefined && this.tiltExpose.property in state) {
```
But as mentioned in another comment, you can probably use a "monitor" for this.